### PR TITLE
candidate-4.2: add 4.2.18

### PR DESCRIPTION
**Pull request by thiagoalessio**

### CODE REVIEW

File: channels/candidate-4.3.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: candidate-4.3
 versions:
 # until s390 is released on 4.3 we may not want to include it in 4.3 channels
 - 4.2.16+amd64
+- 4.2.18
```
Comment (eparis) [inline channels/candidate-4.3.yaml]: Do we want +amd64?

Comment (thiagoalessio) [reply channels/candidate-4.3.yaml]: oh yes, sorry ... added!

**Pull request merged**